scripts/drtprod

```diff
@@ -111,52 +115,94 @@ EOF"
+      - job_name: workload2
+        honor_timestamps: false
+        metrics_path: /metrics
+        tls_config:
+          insecure_skip_verify: true
+        follow_redirects: true
+        enable_http2: true
+        static_configs:
+        - targets:
+          - "localhost:2113"
+        relabel_configs:
+        - action: replace
+          replacement: 2
+          target_label: workload
+      - job_name: workload3
+        honor_timestamps: false
+        metrics_path: /metrics
+        tls_config:
+          insecure_skip_verify: true
+        follow_redirects: true
+        enable_http2: true
+        static_configs:
+        - targets:
+          - "localhost:2114"
+        relabel_configs:
+        - action: replace
+          replacement: 3
+          target_label: workload
 
-instances:
-  - openmetrics_endpoint: http://localhost:2112/metrics
-    namespace: workload1
-    raw_metric_prefix: workload_
-    metrics:
-    - tpcc_.*
-    - kv_.*
-  - openmetrics_endpoint: http://localhost:2113/metrics
-    namespace: workload2
-    raw_metric_prefix: workload_
+processors:
+  filter/workload:
     metrics:
-    - tpcc_.*
-    - kv_.*
-  - openmetrics_endpoint: http://localhost:2114/metrics
-    namespace: workload3
-    raw_metric_prefix: workload_
-    histogram_buckets_as_distributions: true
+      include:
+        match_type: regexp
+        expressions:
+        - workload_tpcc.*
+        - workload_kv_.*
+
+# The */datadog elements are defined in the primary configuration file.
+service:
+  pipelines:
     metrics:
-    - tpcc_.*
-    - kv_.*
+      receivers:
+      - prometheus/workload
+      processors:
+      - memory_limiter/datadog
+      - filter/workload
+      - batch/datadog
+      - attributes/datadog
+      exporters:
+      - datadog
 EOF"
-        roachprod ssh ${cluster} -- 'sudo systemctl enable datadog-agent && sudo systemctl restart datadog-agent'
         ;;
+    esac
 
-      *)
-        roachprod ssh ${cluster} -- "sudo tee /etc/datadog-agent/conf.d/cockroachdb.d/conf.yaml > /dev/null << EOF
----
-init_config:
+    roachprod ssh ${cluster} -- 'sudo systemctl enable datadog-agent && sudo systemctl restart datadog-agent'
 
-instances:
-- openmetrics_endpoint: http://localhost:26258/_status/vars
-  tls_verify: false
-  service: drt-cockroachdb
-EOF"
-        roachprod ssh ${cluster} -- 'sudo systemctl enable datadog-agent && sudo systemctl restart datadog-agent'
+    roachprod opentelemetry-start ${cluster} \
+      --datadog-api-key "${dd_api_key}" \
+      --datadog-tags 'service:drt-cockroachdb,team:drt'
+
+    roachprod fluent-bit-start ${cluster} \
+      --datadog-api-key "${dd_api_key}" \
+      --datadog-service drt-cockroachdb \
+      --datadog-tags 'service:drt-cockroachdb,team:drt'
 
-        roachprod fluent-bit-start ${cluster} \
-          --datadog-api-key "${dd_api_key}" \
-          --datadog-service drt-cockroachdb \
-          --datadog-team drt
-        ;;
-    esac
```

Comment:
Moved the starting of the Datadog agent, Fluent Bit, and the OpenTelemetry Collector out of the `*` case so it runs always. Cleaner than removing the `;;` and falling through the case statement.